spring-data-jdbc/src/main/java/org/springframework/data/jdbc/core/convert/IdGeneratingEntityCallback.java

```diff
@@ -56,14 +56,14 @@ public Object onBeforeSave(Object aggregate, MutableAggregateChange<Object> aggr
 			return aggregate;
 		}
 
-		RelationalPersistentProperty property = entity.getRequiredIdProperty();
+		RelationalPersistentProperty idProperty = entity.getRequiredIdProperty();
```

Comment:
That change was intentional. Please do not change unrelated code.

Reply:
Okay, I'll roll back

Reply:
Yes, please. There's no need to repeat ourselves. There's already `IdGenerating` in the class name and so there's little sense to name variables `id`…. In the end, it becomes noise. Instead, the focus should be on the things that matter within the existing context.